src/web/mjs/connectors/ManHuaGui.mjs

```diff
@@ -14,6 +14,37 @@ export default class ManHuaGui extends SinMH {
         this.pathMatch = '/list/index_p(\\d+).html';
         this.queryMangasPageCount = 'div.pager-cont div.pager a:last-of-type';
         this.queryChapters = 'div.chapter-list ul li a';
+        this.config = {
+            throttle: {
```

Comment:
This is a keyword, the downloader will use this to delay image downloads.
Since the title says `Chapter Throttle` instead of `Image Throttle` this seems not intentional

Reply:
Okay... I'll change the title.